resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+        biigle.$declare('selectedFiles', '{!! collect(old('selected_files',[])) !!}');
+        biigle.$declare('fileLabelIds', '{!! collect(old('only_file_labels',[])) !!}');
+        biigle.$declare('annotationLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('cloneFileLabels', {{old('clone_file_labels',false)}});
+        biigle.$declare('cloneAnnotations', {{old('clone_annotations',false)}});
+        biigle.$declare('annotationLabelIds', '{!! collect(old('only_annotation_labels', [])) !!}');
+        biigle.$declare('cloneUrlTemplate', "{{ url("api/v1/volumes/{$volume->id}/clone-to/:pid") }}")
+    </script>
+@endpush
+
+@section('content')
+    <div class="container">
+        <div class="col-sm-8 col-sm-offset-2 col-lg-6 col-lg-offset-3">
+            <div class="form-group">
+                <h2 class="row">Clone volume "{{$volume->name}}"</h2>
+                <br>
+            </div>
+            <form id="clone-volume-form" class="clearfix" role="form" method="POST" :action="getCloneUrl"
+                  enctype="multipart/form-data" v-on:submit="startLoading" v-on:keydown.enter.prevent>
+                <div class="row">
+                    <div class="form-group">
+                        <label>New volume name</label>
+                        <input type="text" class="form-control" name="name" id="name" v-model="name"
+                               placeholder="My new volume name" ref="nameInput" value="{{old('name')}}" required
+                               autofocus minlength="1" maxlength="512">
+                    </div>
+                    <div class="form-group{{ $errors->has('name') ? ' has-error' : '' }}">
+                        @if($errors->has('name'))
+                            <span class="help-block">{{ $errors->first('name') }}</span>
+                        @endif
+                    </div>
+
+                    <div class="form-group">
+                        <label>New volume destination project</label>
+                        <typeahead class="typeahead--block" :items="getProjects"
+                                   placeholder="Select destination project"
+                                   title="Select project to clone volume to"
+                                   v-on:select="setProject" :clear-on-select="false"
+                                   :value="setDefaultProject" required></typeahead>
+                    </div>
+
+                    <div class="checkbox">
+                        <label><input type="checkbox" id="files" v-model="cloneFiles">
+                            Filter <span>
+                            @if($volume->isImageVolume())
+                                    images
+                                @else
+                                    videos
+                                @endif
+                            </span>
+                        </label>
+                    </div>
+                    <div v-if="cloneFiles">
```

Comment:
This element "flashes" when the page loads, i.e. it is briefly shown until Vue is initialized to hide it again. Use [`v-cloak`](https://vuejs.org/api/built-in-directives.html#v-cloak) to hide the element from the beginning:

```suggestion
                    <div v-if="cloneFiles" v-cloak>
```